src/screens/SignInV2/elements/SlideShow.js

```diff
@@ -35,19 +35,19 @@ const SlideShow = ({handleLogin, onContainerPress = () => {}}) => {
     },
     {
       illustrations: FgOnboarding2,
-      title: 'Receive Incognito Messages',
+      title: 'Better Karma, Higher Visibility',
       text: (
         <Text>
-          {'Share your link on other platforms to receive'}
+          {'Karma scores help the community stay'}
           {'\n'}
 
-          {'messages from your friends. Ask for advice,'}
+          {'accountable even when in Incognito Mode.'}
           {'\n'}
 
-          {'feedback, and more!'}
+          {'To increase your score, just invite followers,'}
           {'\n'}
 
-          {'As always, abuse can be blocked in two clicks.'}
+          {'create posts and avoid being blocked!'}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new text provides a clear explanation of how users can increase their karma scores. It's more informative than the previous version and gives users actionable steps to improve their visibility within the community.
```diff
-      title: 'Receive Incognito Messages',
-       text: (
-         <Text>
-          {'Share your link on other platforms to receive'}
-           {'\n'}
- 
-          {'messages from your friends. Ask for advice,'}
-           {'\n'}
- 
-          {'feedback, and more!'}
-           {'\n'}
- 
-          {'As always, abuse can be blocked in two clicks.'}
-         </Text>
-       )
+      title: 'Better Karma, Higher Visibility',
+       text: (
+         <Text>
+          {'Karma scores help the community stay'}
+           {'\n'}
+  
+          {'accountable even when in Incognito Mode.'}
+           {'\n'}
+  
+          {'To increase your score, just invite followers,'}
+           {'\n'}
+  
+          {'create posts and avoid being blocked!'}
+         </Text>
+       )
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->